pkg/driver/vz/vz_driver_darwin.go

```diff
@@ -246,3 +265,44 @@ func (l *LimaVzDriver) GuestAgentConn(_ context.Context) (net.Conn, error) {
 	}
 	return nil, errors.New("unable to connect to guest agent via vsock port 2222")
 }
+
+func (l *LimaVzDriver) Name() string {
+	return "vz"
+}
+
+func (l *LimaVzDriver) Register(_ context.Context) error {
+	return nil
+}
+
+func (l *LimaVzDriver) Unregister(_ context.Context) error {
+	return nil
+}
+
+func (l *LimaVzDriver) ChangeDisplayPassword(_ context.Context, _ string) error {
+	return nil
+}
+
+func (l *LimaVzDriver) GetDisplayConnection(_ context.Context) (string, error) {
+	return "", nil
+}
+
+func (l *LimaVzDriver) CreateSnapshot(_ context.Context, _ string) error {
+	return errors.New("unimplemented")
+}
+
+func (l *LimaVzDriver) ApplySnapshot(_ context.Context, _ string) error {
+	return errors.New("unimplemented")
+}
+
+func (l *LimaVzDriver) DeleteSnapshot(_ context.Context, _ string) error {
+	return errors.New("unimplemented")
+}
+
+func (l *LimaVzDriver) ListSnapshots(_ context.Context) (string, error) {
+	return "", errors.New("unimplemented")
```

Comment:
The common `ErrImplemented` should be defined so that it can be compared with `errors.Is`